### PR TITLE
Added Locathah Race from Locathah Rising

### PR DESCRIPTION
**Pull request by Azzzzzy**
https://www.dmsguild.com/product/289261/Locathah-Rising-5e

### CODE REVIEW

File: supplements/ghosts-of-saltmarsh/races/race-locathah.xml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+			<set name="url">https://www.dmsguild.com/product/289261/Locathah-Rising-5e</set>
+			<set name="image">https://www.dmsguild.com/images/44/289261.jpg</set>
+			<set name="author" abbreviation="WOTC" url="http://dnd.wizards.com">Wizards of the Coast</set>
+			<set name="official">true</set>
+			<set name="supplement">true</set>
+			<set name="release">20190915</set>
+		</setters>
+	</element>
+
+    <element name="Locathah" type="Race" source="Locathah Rising" id="ID_WOTC_GOS_RACE_LOCATHAH">
```
Comment (swdriessen) [inline supplements/ghosts-of-saltmarsh/races/race-locathah.xml]: You can change the `GOS` in the IDs to `LR`

File: supplements/ghosts-of-saltmarsh/races/race-locathah.xml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            <stat name="dexterity" value="1" />
+            <stat name="innate speed" value="30" bonus="base"/>
+            <stat name="innate speed:swim" value="30" bonus="base" />
+            <grant type="Size" id="ID_SIZE_MEDIUM" />
+
+            <grant type="Language" id="ID_LANGUAGE_COMMON" />
+            <grant type="Language" id="ID_MM_LANGUAGE_AQUAN" />
+            <grant type="Racial Trait" id="ID_WOTC_GOS_RACIAL_TRAIT_LOCATHAH_OBSERVANT_AND_ATHLETIC" />
+            <grant type="Racial Trait" id="ID_WOTC_GOS_RACIAL_TRAIT_LOCATHAH_LEVIATHAN_WILL" />
+            <grant type="Racial Trait" id="ID_WOTC_GOS_RACIAL_TRAIT_LOCATHAH_NATURAL_ARMOR" />
```
Comment (swdriessen) [inline supplements/ghosts-of-saltmarsh/races/race-locathah.xml]: You can bump the natural armor up to keep the order of the features.

**Comment by swdriessen** [comment]
oh, and you can add the names from the sheets as race names: `Barnacle, Bubbles, Gl'ulp, Sandlot, Shanty, Undertow`

**Comment by Azzzzzy** [comment]
Should be good now!

**Comment by swdriessen** [comment]
Thanks!

fixed natural armor (dex mod) and created separate index

**Pull request merged**